frontend/src/components/header/MobileNav.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Box,
+  Text,
+  Stack,
+  useColorModeValue,
+  useDisclosure,
+  Flex,
+  MenuList,
+  Menu,
+  MenuButton,
+  Button,
+  MenuDivider,
+} from "@chakra-ui/react";
+import { FC } from "react";
+import { NAV_ITEMS, NavItem } from "./DesktopNav";
+import { AuthButtons } from "../authentication/SignInOrOut";
+import { CloseIcon, HamburgerIcon } from "@chakra-ui/icons";
+
+const MobileNavItem = ({ label, children, href }: NavItem) => {
+  const { onToggle } = useDisclosure();
+
+  return (
+    <Stack spacing={4} onClick={children && onToggle}>
+      <Box
+        py={2}
+        as="a"
+        href={href ?? "#"}
+        justifyContent="space-between"
+        alignItems="center"
+        textAlign={"center"}
+        _hover={{
+          textDecoration: "none",
+        }}
+      >
+        <Text
+          fontWeight={600}
+          color={useColorModeValue("gray.600", "gray.200")}
+          fontSize={"lg"}
+        >
+          {label}
+        </Text>
+      </Box>
+    </Stack>
+  );
+};
+
+interface NavBarProps {}
```

Comment:
Whats this for?

Reply:
Style and pizzazz!!.....

It was a fundamental misunderstanding of your request for me to use FC :) I'll prune it